MoltenVK/MoltenVK/Vulkan/vulkan.mm

```diff
@@ -492,8 +492,15 @@ MVK_PUBLIC_VULKAN_SYMBOL VkResult vkMapMemory(
    void**                                      ppData) {
 
 	MVKTraceVulkanCallStart();
+	VkMemoryMapInfoKHR mapInfo = {};
+	mapInfo.sType = VK_STRUCTURE_TYPE_MEMORY_MAP_INFO_KHR;
+	mapInfo.flags = flags;
+	mapInfo.memory = mem;
+	mapInfo.offset = offset;
+	mapInfo.size = size;
```

Comment:
I'd prefer you set `pNext` to `nullptr` explicitly, rather than relying on the `={}`, so we are deliberate about it being set to a valid value.